modules/user/user.resolver.js

```diff
@@ -1,20 +1,31 @@
 const userService = require('./user.service');
 
+const Unauthorized = `${[
+  {
+    lang: 'uk',
+    value: `Користувач неавторизований`,
```

Comment:
Translated messages should match in meaning, grammar and count of words as much as possible

Reply:
I think it is not the best idea to put an array inside the template string. Can we pass an array inside `AuthenticationError`? Because if we pass a string it serializes array to something like this:

Reply:
fixed